local/bin/py/build/configurations/pull_config_preview.yaml

```
@@ -268,6 +268,20 @@
           title: Getting Started with Android RUM Collection
           kind: documentation
 
+  - repo_name: browser-sdk
+    contents:
+    - action: pull-and-push-file
+      branch: ruth/docs-1217
```

```suggestion
      branch: master
```
Needs to be updated right before merge